sdk/eventhub/Microsoft.Azure.EventHubs/src/Microsoft.Azure.EventHubs.csproj

```diff
@@ -28,9 +28,9 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Amqp" />
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" />
-    <PackageReference Include="System.Diagnostics.DiagnosticSource" />
-    <PackageReference Include="System.Net.Http" />
-    <PackageReference Include="System.Reflection.TypeExtensions" />
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Condition="'$(TargetFramework)' != 'net6.0'" />
```

Comment:
`Microsoft.Azure.EventHubs` is a deprecated package owned by the service team; it is only receiving security updates and high priority bug fixes.     Please revert the changes here and to the targets that would impact it.

Reply:
So don't change the deprecated packages?